dockers/base-ipu/Dockerfile

```diff
@@ -37,59 +37,49 @@ RUN apt-get update -qq && \
         libopenmpi-dev \
         unzip \
         wget \
+        software-properties-common \
+    && \
+    # Install python
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get install -y \
+        python${PYTHON_VERSION} \
+        python${PYTHON_VERSION}-distutils \
+        python${PYTHON_VERSION}-dev \
     && \
-# Install conda and python.
-# NOTE new Conda does not forward the exit status... https://github.com/conda/conda/issues/8385
-    curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py39_${CONDA_VERSION}-Linux-x86_64.sh && \
-    chmod +x ~/miniconda.sh && \
-    ~/miniconda.sh -b && \
-    rm ~/miniconda.sh && \
-# Cleaning
+    update-alternatives --install /usr/bin/python${PYTHON_VERSION%%.*} python${PYTHON_VERSION%%.*} /usr/bin/python${PYTHON_VERSION} 1 && \
```

Comment:
Not sure if all HPU drivers are fully compatible with any Python version

Reply:
IMO, having it as a variable itself is fine.